NYPLCardCreator/Flow/UserSummaryViewController.swift

```diff
@@ -302,24 +279,32 @@ final class UserSummaryViewController: TableViewController, JuvenileCardCreation
           comment: "An alert message explaining an error and telling the user to try again later"))
```

Comment:
in the previous line it seems like the error message is the default already included inside the `showErrorAlert` function, so it could be omitted. Or we could provide a more custom message.

Reply:
Right, removed the message, I think a generic message is enough in this case.